onnxruntime/test/providers/cpu/nn/conv_transpose_op_test.cc

```diff
@@ -56,8 +63,80 @@ void TestConvTransposeOp(const ConvTransposeOpAttributes& attributes,
 
   test.Run(expect_result, err_str, excluded_provider_types);  // Disable TensorRT because weight as input is not supported
 }
+
 }  // namespace
 
+TEST(ConvTransposeTest, ConvTranspose_1D_AsymmetricPads) {
```

Comment:
Builds are failing on account of the added tests as something in the CPU implementation is unable to handle asymmetric padding correctly (CUDA by definition can't handle asymmetric padding). Need to fix this issue before these tests will pass.